File: text/R/create/text_create.R
# ---------------------------------------------------------
#	text_create.R
#
#				Feb/21/2014
#
# ---------------------------------------------------------
aa=paste("t2381","名古屋","47825","1946-8-24",sep="\t")
bb=paste("t2382","豊橋","28157","1946-11-25",sep="\t")
cc=paste("t2383","岡崎","74192","1946-9-2",sep="\t")
dd=paste("t2384","一宮","38754","1946-6-25",sep="\t")
ee=paste("t2385","蒲郡","46352","1946-10-14",sep="\t")
ff=paste("t2386","常滑","65218","1946-9-12",sep="\t")
gg=paste("t2387","大府","37241","1946-3-21",sep="\t")
hh=paste("t2388","瀬戸","53496","1946-7-26",sep="\t")
ii=paste("t2389","犬山","26157","1946-12-2",sep="\t")


print ("*** 開始 ***")
file_out <- commandArgs(trailingOnly=TRUE)[2]
print (file_out)

write (aa,file=file_out)
write (bb,file=file_out, append=T)
write (cc,file=file_out, append=T)
write (dd,file=file_out, append=T)
write (ee,file=file_out, append=T)
write (ff,file=file_out, append=T)
write (gg,file=file_out, append=T)
write (hh,file=file_out, append=T)
write (ii,file=file_out, append=T)

print ("*** 終了 ***")
# ---------------------------------------------------------

File: text/elixir/create/text_create.exs
# ------------------------------------------------------------------
#
#	text_create.exs
#
#						Jul/24/2015
# ------------------------------------------------------------------
defmodule MyModule do
    def write(file_out, word ) do
        case File.open file_out, [:write] do          # 開く
            {:ok, file} -> writeStr( file, word )    # 書き込む
                           File.close file           # 閉じる
            {:error, posix } -> IO.puts posix
        end
    end
 
    def writeStr( file, word ) do
        IO.binwrite file, word
    end
end

# ------------------------------------------------------------------
IO.puts "*** 開始 ***"
file_out = hd System.argv()
IO.puts file_out
aa = "t2381\t名古屋\t93514\t1910-5-17\n"
bb = "t2382	豊橋	31298	1910-9-10\n"
cc = "t2383	岡崎	28458	1910-2-14\n"
dd = "t2384	一宮	62542	1910-7-9\n"
ee = "t2385	蒲郡	42619	1910-8-4\n"
ff = "t2386	常滑	35182	1910-1-21\n"
gg = "t2387	大府	81247	1910-7-23\n"
hh = "t2388	瀬戸	25798	1910-10-26\n"
ii = "t2389	犬山	84135	1910-12-15\n"
str_out = Enum.join([aa,bb,cc,dd,ee,ff,gg,hh,ii])
MyModule.write(file_out,str_out)
IO.puts "*** 終了 ***"
# ------------------------------------------------------------------
